pulp_container/tests/unit/test_serializers.py

```diff
@@ -21,6 +24,14 @@ def setUp(self):
         self.push_repository_href = "/pulp/api/v3/repositories/container/container-push/{}/".format(
             self.push_repository.pk
         )
+        self.user = get_user_model().objects.create(username="user1", is_staff=False)
+        _set_current_user(self.user)
+
+    def tearDown(self):
+        """Delete the user."""
+        super(TestContainerDistributionSerializer, self).tearDown()
```

Comment:
```suggestion
        super().tearDown()
```
Welcome to world of python 3! ;)

Reply:
Thank you!